src/main/java/marquez/api/resources/JobResource.java

```diff
@@ -149,12 +151,25 @@ public Response create(
         .build();
   }
 
-  @Timed
-  @ResponseMetered
-  @ExceptionMetered
   @GET
-  @Path("/jobs/runs/{id}")
   @Produces(APPLICATION_JSON)
+  @Path("/namespaces/{namespace}/jobs/{job}/runs")
```

Comment:
We'll also want to add the following metric annotations:

```java
@Timed
@ResponseMetered
@ExceptionMetered
@GET
@Path("/namespaces/{namespace}/jobs/{job}/runs")
@Produces(APPLICATION_JSON)
public Response getRunsForJob(...) {
  ...
}
```

Reply:
totally missed this good catch